test/lint/commit-script-check.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/env sh
```

Comment:
./ci/lint/06_script.sh: test/lint/commit-script-check.sh: /bin/env: bad interpreter: No such file or directory

Reply:
I made changes according to the developer notes here: https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#shebang

Reply:
It should be /usr/bin/env instead of /bin/env

Reply:
I made changes according to developer notes here: https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#shebang

Reply:
Missing the `/usr` -- not everyone has merged `/usr`. If  you're changing to to env, probably should also change it to bash as well to avoid getting weird incompatibilities?

Reply:
I changed it back to #!/bin/sh in a later commit